Makefile

```diff
@@ -27,13 +28,13 @@ check-dummy:
 
 try-runtime-upgrade-battery-station:
 	@$(MAKE) TRYRUNTIME_URL="wss://bsr.zeitgeist.pm:443" \
-		RUNTIME_PATH="./target/debug/wbuild/battery-station-runtime/battery_station_runtime.compact.compressed.wasm" \
+		RUNTIME_PATH="./target/release/wbuild/battery-station-runtime/battery_station_runtime.compact.compressed.wasm" \
 		-- \
 		--try-runtime
 
 try-runtime-upgrade-zeitgeist:
 	@$(MAKE) TRYRUNTIME_URL="wss://zeitgeist-rpc.dwellir.com:443" \
-		RUNTIME_PATH="./target/debug/wbuild/zeitgeist-runtime/zeitgeist_runtime.compact.compressed.wasm" \
+		RUNTIME_PATH="./target/release/wbuild/zeitgeist-runtime/zeitgeist_runtime.compact.compressed.wasm" \
```

Comment:
I think the reason for using debug was more verbose logs. Why release? Why additional cargo build --release --features=parachain,try-runtime ? If I was using `make try-runtime-upgrade-zeitgeist`, it always built the node itself. Why?

Reply:
Logs should work the same regardless of whether debug symbols are in the binary. Choosing release here as this is the "official" recommendation. If you want to gauge runtime complexity, I think it makes sense to have something closer to production.

Reply:
> If I was using make try-runtime-upgrade-zeitgeist, it always built the node itself. Why?